src/util/files/resource_loader.c

```diff
@@ -3,23 +3,22 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-
-//TODO: REMOVE THIS IN THE FUTURE, ONLY FOR TESTING
-//TODO: REPLACE ALL MALLOC WITH CALLOC
 #include <limits.h>
```

Comment:
Hi Adam,

Please can you put these includes in alphabetical order

Kind regard,
Toby